### PR TITLE
java convenction changes

### PR DESCRIPTION
**Pull request by RANJEETJ06**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

File: src/main/java/com/thealgorithms/datastructures/trees/GenericTree.java [hunk 1]
```diff
@@ -16,24 +16,24 @@
...
 
-    private Node root;
+    private final Node root;
     public GenericTree() { // Constructor
         Scanner scn = new Scanner(System.in);
         root = create_treeG(null, 0, scn);
     }
 
-    private Node create_treeG(Node node, int childindx, Scanner scn) {
+    private Node create_treeG(Node node, int childIndx, Scanner scn) {
```
Comment (siriak) [inline src/main/java/com/thealgorithms/datastructures/trees/GenericTree.java:31]:
```suggestion
    private Node create_treeG(Node node, int childIndex, Scanner scanner) {
```

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Comment by siriak** [comment]
The build is failing because you forgot to update usage of parameters in the function

**Comment by RANJEETJ06** [comment]
how to fix it i don't know

**Review by RANJEETJ06 — COMMENTED** [review]
java convenction changes

**Review by debasishbsws — APPROVED** [review]
Looks Good Thanks for you contribution.

**Pull request merged**